SafeAccountsAPI/Program.cs

```diff
@@ -1,15 +1,19 @@
 ﻿using System;
+using System.IO;
```

Comment:
I dont think we need either of the new using statements, because no code was added